tools/docs/github-query-items.ts

```diff
@@ -1,16 +1,157 @@
-export type GithubApiQueryResponse = {
+import { DateTime } from 'luxon';
+import { logger } from '../../lib/logger';
+import * as hostRules from '../../lib/util/host-rules';
+import { GithubHttp } from '../../lib/util/http/github';
+import { getQueryString } from '../../lib/util/url';
+
+const gitHubApiUrl = 'https://api.github.com/search/issues?';
+const githubApi = new GithubHttp();
+
+if (process.env.GITHUB_TOKEN) {
+  logger.debug('Using GITHUB_TOKEN from env');
+  hostRules.add({
+    matchHost: 'api.github.com',
+    token: process.env.GITHUB_TOKEN,
+  });
+}
+
+type GithubApiQueryResponse = {
   total_count: number;
   incomplete_results: boolean;
   items: ItemsEntity[];
 };
 
-export type ItemsEntity = {
+type ItemsEntity = {
```

Comment:
```suggestion
export type ItemsEntity = {
```
exported via `Items` below